experimental/dds/tree2/src/test/simple-tree/proxies.spec.ts

```diff
@@ -19,9 +19,7 @@ describe("SharedTree proxies", () => {
 	const schema = sb.object("parent", {
 		object: childSchema,
 		list: sb.array(sb.number),
-		// Is there a way to avoid the cast to 'any' when using 'class-schema'?
-		// TODO: https://dev.azure.com/fluidframework/internal/_workitems/edit/6551
-		map: sb.map("map", sb.optional(sb.string) as any),
+		map: sb.map("map", sb.string),
```

Comment:
Maps are not currently working with an '_.optional(..)' value type.  (AB#6551)

Attempting to work around this via an 'any' cast was causing the "cache and reuse" tests below to fail during initialization (AB#6557).

This fixes the issue by avoiding the use of '_.optional', which should not affect the outcome of the test.

Reply:
Are maps intended to support optional values at the schema level? Map values are inherently optional by nature of the key/value pairs being optional. I seem to remember there being a design reason for explicitly not allowing optional map values (something about all key/values pairs implicitly existing, so an optional value associated with a key is equivalent to a key pointing to no value).

Reply:
@Josmithr - That's a really good insight.  You're right that this may be the internal model leaking through.